src/main/java/net/thewyvern/webapp/ec2/config/RedisConfig.java

```diff
@@ -6,27 +6,32 @@
 import java.io.InputStream;
 import java.util.Properties;
 
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Profile;
 import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
 import org.springframework.session.data.redis.config.ConfigureRedisAction;
 import org.springframework.session.data.redis.config.annotation.web.http.EnableRedisHttpSession;
 
 import redis.clients.jedis.JedisPoolConfig;
 
 /**
- * 
- * @author matthewd
- *
  * Note that elisticache redis isn't quite redis...
  * 
  * See: https://github.com/spring-projects/spring-session/issues/97
  *      https://github.com/spring-projects/spring-session/issues/124
  *
+ * Notes:
+ * - Build project with -Predis for this to work...
  */
-
```

Comment:
For local, you can also do "./run-tomcat-local.sh redis" - first param is considered Spring profiles.